packages/web3-shared/evm/package.json

```diff
@@ -10,6 +10,8 @@
   },
   "sideEffects": false,
   "dependencies": {
+    "@codingsh/harmony-crypto": "0.1.58",
+    "@harmony-js/utils": "^0.1.56",
```

Comment:
cc @guanbinrui

Reply:
I will remove these packages that are not being used